src/main/java/com/example/auth/dao/UserDAO.java

```diff
@@ -0,0 +1,60 @@
+
+package com.example.auth.dao;
+
+import com.example.auth.model.User;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class UserDAO {
+
+    private final Connection conn;
+
+    // Constructor to initialize with a Connection object
```

Comment:
rather create connection inside every dao method before access. It is better why ask GPT?